### PR TITLE
Plugin dll load method change.

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Currently neo load plugin dll file directly, which will lock dll files and result in "uninstall" command fails.
So I change load method to load a copied dll file to resolve this issue.

### CODE REVIEW

**Comment by vncoelho** [comment]
There is something strange...I try to install from version `version 2.10.2.0` and it tries to download.

```cs
Downloading from ***v2.10.2.1.zip
error
```

**Comment by vncoelho** [comment]
Versions of neo-plugins are latests neo-plugins are different, 2.10.2.0 and  2.10.2.1.

Fell free to merge @shargon , we need another PR for extra adjustments, maybe neocli should optionally read both: plugin name and, optionally, version

File: neo/Plugins/Plugin.cs [hunk 1]
```diff
@@ -97,7 +97,8 @@ internal static void LoadPlugins(NeoSystem system)
             if (!Directory.Exists(pluginsPath)) return;
             foreach (string filename in Directory.EnumerateFiles(pluginsPath, "*.dll", SearchOption.TopDirectoryOnly))
             {
-                Assembly assembly = Assembly.LoadFile(filename);
+                var file = File.ReadAllBytes(filename);
+                Assembly assembly = Assembly.Load(file);
```
Comment (igormcoelho) [inline neo/Plugins/Plugin.cs]: Very interesting solution... since sizes are very low, it shouldn't impact memory.

**Comment by vncoelho** [comment]
@Ashuaidehao, could you port this to master branch?

**Comment by Ashuaidehao** [comment]
> @Ashuaidehao, could you port this to master branch?

Sure ,I have created another PR(https://github.com/neo-project/neo/pull/816/) port to master branch.

**Pull request merged**